src/components/AddExpensesForm.js

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import { v4 as uuid } from 'uuid';
+
+const AddExpensesForm = () => {
+  const initialState = {
+    id: '',
+    name: '',
+    date: '',
+    amount: '',
+  };
+
+  const [singleExpense, setSingleExpense] = useState(initialState);
+  const [expenses, setExpenses] = useState([]);
+
+  const handleChange = (evt) => {
+    // easy way to take any input name and assign its value to the initialState as long as they use the same key (e.g. <input name="amount"/> => singleExpense.amount)
+    const { name, value } = evt.target;
```

Comment:
Easy way to take any `input` `name` and assign its `value` to the `initialState` as long as they use the same `key` (e.g. `<input name="amount"/>` => `singleExpense.amount`)

Reply:
sorry Matt I resolved this by mistake. I renamed `name` to `title`